package.json

```diff
@@ -18,32 +18,33 @@
   "main": "./dist/index.js",
   "types": "./dist/index.d.ts",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "jest --no-cache",
     "lint": "eslint --max-warnings=0 --ext .ts,.tsx,.js,.jsx src",
     "prettycheck": "prettier -c 'src/**/*.{ts,js,css,html}'",
     "prettyfix": "prettier --write 'src/**/*.{ts,js,css,html}'",
     "compile": "tsc",
+    "build": "tsc",
```

Comment:
we already have compile.

Reply:
but I always typed build, so I added it :)

Reply:
we will bring this up in next meeting (to have on or another), but for now please remove it.